README.md

```diff
@@ -23,3 +23,40 @@ gradle bootRun --refresh-dependencies
+| IEXEC_WORKER_TEE_COMPUTE_MAX_HEAP_SIZE_GB     | Max heap size for TEE apps.                                                                                                      | Positive integer | 8                                                         |
+| IEXEC_WORKER_DOCKER_NETWORK_NAME              | Name of the network the LAS is in. Useless if worker runs only standard tasks.                                                   | String           | iexec-worker-net                                          |
+| IEXEC_WORKER_DOCKER_REGISTRY_USERNAME_0       | Username to pull apps from [docker official website](https://hub.docker.com/)                                                    | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_PASSWORD_0       | Password to  pull apps from [docker official website](https://hub.docker.com/)                                                   | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_ADDRESS_1        | Secondary registry address.                                                                                                      | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_USERNAME_1       | Username to pull apps from secondary registry.                                                                                   | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_PASSWORD_1       | Password to pull apps from secondary registry.                                                                                   | String           |                                                           |
+| IEXEC_WORKER_WALLET_PATH                      | Path to the wallet that should be used by the worker.                                                                            | String           | ./src/main/resources/wallet/encrypted-wallet_worker1.json |
+| IEXEC_WORKER_WALLET_PASSWORD                  | Password of the wallet that should be used by the worker.                                                                        | String           | whatever                                                  |
+| IEXEC_SCONE_SHOW_VERSION                      | Whether to display version of Scone.                                                                                             | Boolean          | true                                                      |
+| IEXEC_SCONE_LOG_LEVEL                         | Log level Scone should use.                                                                                                      | # TO COMPLETE #  | debug                                                     |
```

Comment:
I haven't found any concrete info on Scone log level so can't complete this part myself.

Reply:
https://sconedocs.github.io/SCONE_ENV/#debug :
_SCONE_LOG set to value between 0 (no) and 7 (debug) to see more or less messages on stderr from the SCONE platform. Messages could be warnings if certain functions are not (yet) implemented inside of an enclave._
String values alternative got to be somewhere.

Reply:
Thanks, I'll add that info.

Reply:
Nice work for the properties.
Maybe add that properties are `iexec.core.port` and can be overriden with environment variables `IEXEC_CORE_PORT` (uppercases with `.` replaced with `_`)

Reply:
That's not always true so it can't be a general rule. But I can add a new column if you find it useful - or simply rename `Property` into `Env var`.

Reply:
Hmm, I don't know. See how the spring boot documentation is written.
https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html

Reply:
I think Spring Boot is different than our project. Spring Boot is a framework which we interact with in our code (= properties). iExec Worker is more like an app people interact with through env vars.

Reply:
I don't understand what you mean by **Spring Boot is different from our project**.
Spring Boot is a framework and we build applications upon it.
The documentation I provided show the available properties defined by the framework and available at the application level, given the related spring boot modules have been enabled/imported.
For further reading on Spring Boot applications properties, you can read:
* https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html
* https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.relaxed-binding
* https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.relaxed-binding.environment-variables
* https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config

In my understanding, as the environment variables naming convention is rather strict and well adopted, there is a kind of consensus to use another notation for properties.